admin/controllers/pupil-pin.js

```diff
@@ -154,10 +154,11 @@ const postGeneratePins = async function postGeneratePins (req, res, next) {
     const update = pinGenerationService.generateSchoolPassword(school)
     if (update) {
       await schoolDataService.sqlUpdate(R.assoc('id', school.id, update))
+      school.pin = update.pin
     }
 
     // depends on school pin being ready
-    await checkStartService.prepareCheck2(pupilsList, req.user.School, req.user.schoolId, isLiveCheck, school.timezone)
+    await checkStartService.prepareCheck2(pupilsList, school, isLiveCheck)
```

Comment:
@danwilliams does the whole school record required to be passed down?
Would it be best to provide a destructured format with just the props that are needed on `prepareCheck2` method?